addons/mail/static/src/js/client_action.js

```diff
@@ -679,6 +801,61 @@ var ChatAction = Widget.extend(ControlPanelMixin, {
         this.domain = result.domain;
         this.fetch_and_render_thread();
     },
+
+    /**
+     * @private
+     * @param {MouseEvent}
+     */
+    _onMobileTabClick: function (event) {
```

Comment:
handlers should be ordered alphabetically (in the handlers section), so this should go above _onSearch